doc/manual/typing.md

```diff
@@ -255,6 +255,26 @@ The following type constructors are available:
   }
   ```
 
+### Subtyping
+
+Subtyping is a relation between two types where
+you can use one of the type in place of the other one.
+In Nickel, there is :
+
+- **Record/Dictionary** subtyping : `{field1 : T1, ..., fieldn : Tn} <= { _ : T}`
```

Comment:
Usually it's customary to use the symbol `<:`. It's also better to define it before (after the informal intro, you can say that it's call "subtyping" and we write `T <: U` if a value of type `T` can be freely used as a value of type `U`.

```suggestion
- **Record/Dictionary** subtyping : `{field1 : T1, ..., fieldn : Tn} <: { _ : T}`
```